.github/workflows/appsec.yml

```diff
@@ -0,0 +1,47 @@
+name: AppSec
```

Comment:
Can we sync together on the appsec tests ? maybe there is a quick way to make them work with this PR instead of putting them aside.

Reply:
Sure, although we should probably split the tests by functionalities anyway.

Reply:
I would say yes since it makes it easier to isolate which functionality is passing or failing.